eos-parental-controls-client/eos-parental-controls-client.py

```diff
@@ -164,6 +192,18 @@ def main():
     parser_check.add_argument('path',
                               help='path to a program to check')
 
+    # ‘oars-section’ command
+    parser_oars_section = subparsers.add_parser('oars-section',
+                                                parents=[common_parser],
+                                                help='get the value of a '
+                                                     'given OARS section')
+    parser_oars_section.set_defaults(function=command_oars_section)
+    parser_oars_section.add_argument('user', default='', nargs='?',
+                                     help='user ID or username to get the '
+                                          'OARS filter for (default: current '
+                                          'user)')
+    parser_oars_section.add_argument('section', help='OARS section to get')
+
     # Parse the command line arguments and run the subcommand.
     args = parser.parse_args()
     args_dict = dict((k, v) for k, v in vars(args).items() if k != 'function')
```

Comment:
```python
args_dict = vars(args)
del args_dict['function']
```

Reply:
What’s the advantage to doing it this way? I kind of prefer the one-step production of `args_dict`, rather than a two-step production and modification. That said, the comprehension is quite verbose.

Reply:
Just the verbosity!